internal/common/objects/BaseBiker.go

```diff
@@ -269,6 +270,26 @@ func (bb *BaseBiker) FinalDirectionVote(proposals []uuid.UUID) voting.LootboxVot
 	return votes
 }
 
+func (bb *BaseBiker) VoteForKickout(agentsOnBike []IBaseBiker) uuid.UUID {
```

Comment:
I think this function should allow for people to vote more than one agents off (or zero)
So should probably return either a list if we don't want to include a ranking in who you vote off, or a map if we want to experiment with different kinds of voting